lib/wallet/wallet.js

```diff
@@ -3267,7 +3298,15 @@ class Wallet extends EventEmitter {
   }
 
   /**
-   * Input size estimator for max possible tx size.
+   * Estimate witness size given output address.
+   * Unlike Bitcoin, our signatures are always 65 bytes.
+   * However, we still assume that the witness varInt size
+   * is only one byte. In short, this estimate may be off
+   * by 1 (at most) but only if a witness has > 253 items.
```

Comment:
...actually if the witness stack is too big, this varInt will be 3 bytes so off by 2 at most.